HotelBooking/Views/reserved_rooms_admin_view.py

```diff
@@ -5,6 +5,15 @@
 from typing import Tuple, List
 from PyInquirer import prompt
 from HotelBooking.Models.room import ROOM_TYPE
+from datetime import datetime
+
```

Comment:
Not a comment but a complaint. 

LMAO who named this file? Why does it have such a weird name? Do we have a `reserved_rooms_customer_view` then?

Reply:
lolll didn't you make the admin stuff:')

Reply:
I'm pretty sure I made a file named `reserved_rooms_view`. Someone else split the difference.

Reply:
Doesn't matter I guess, leave it be.